maykin_2fa/forms.py

```diff
@@ -0,0 +1,13 @@
+from django.core.exceptions import ValidationError
+
+from two_factor.forms import BackupTokenForm
+
+
+class CustomBackupTokenForm(BackupTokenForm):
+    def clean_otp_token(self):
+        otp_token = self.cleaned_data.get("otp_token")
+
+        if otp_token == "invalid-code":
+            raise ValidationError("Invalid token", code="invalid_token")
+
+        return otp_token
```

Comment:
I think this triggers if the actual backup token you enter is `invalid-code`, which leads to duplicate error messages:

![afbeelding](https://github.com/user-attachments/assets/64ebba0e-fe54-48f7-8d09-51dbdbaf49a6)

I don't think we need the custom form, just the changes to the template to show the errors
```suggestion
```